server/models/schema.sql

```diff
@@ -1564,6 +1564,8 @@ CREATE TABLE `invoice` (
   `description`         TEXT NOT NULL,
   `created_at`          TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
   `is_distributable`    TINYINT(1) NOT NULL ,
+  `is_credit_note`      TINYINT(1),
+  `credit_note_by`      SMALLINT(5) UNSIGNED,
```

Comment:
This information should be stored in the posting journal.  Why do you need it in this table?